### PR TITLE
Add 0.0.22 updates

### PR DESCRIPTION
**Pull request by nicoloboschi**
* Mini-langstream
* ai-text-completions

### CODE REVIEW

File: configuration-resources/large-language-models-llms/vertex-configuration.md [hunk 1]
```diff
@@ -1,21 +1,29 @@
...
       name: "Google Vertex AI configuration"
       configuration:
-        url: "{{ secrets.vertex-ai.url }}"
-        # use triple quotes in order to turn off escaping
-        serviceAccountJson: "{{{ secrets.vertex-ai.serviceAccountJson }}}"
+        url: "https://us-central1-aiplatform.googleapis.com"
+        region: "us-central1"
+        project: "my-gcp-project"
+        # Authentication is done either via service account JSON
+        serviceAccountJson: "{\"type\":..........}"
```
Comment (eolivelli) [inline configuration-resources/large-language-models-llms/vertex-configuration.md:20]: it is better to use a secrets here, and show how to use the <file:xxx> syntax in the secrets (you can also add a reference to the page in which we talk about <file:xxx>

File: SUMMARY.md [hunk 2]
```diff
@@ -47,9 +47,9 @@
 ## Integrations
 
 * [Large Language Models (LLMs)](configuration-resources/large-language-models-llms/README.md)
-  * [open-ai-configuration](configuration-resources/large-language-models-llms/open-ai-configuration.md)
-  * [hugging-face-configuration](configuration-resources/large-language-models-llms/hugging-face-configuration.md)
-  * [vertex-configuration](configuration-resources/large-language-models-llms/vertex-configuration.md)
+  * [OpenAI](configuration-resources/large-language-models-llms/open-ai-configuration.md)
+  * [Hugging Face Inference](configuration-resources/large-language-models-llms/hugging-face-configuration.md)
```
Comment (eolivelli) [inline SUMMARY.md:51]: I would drop "inference", the Embeddings Service is not "inference", is it ?

File: configuration-resources/large-language-models-llms/vertex-configuration.md [hunk 3]
```diff
@@ -1,21 +1,29 @@
 # vertex-configuration
 
-A Google vertex configuration.
+A vertex-configuration resource is used for AI agents.
+
+{% hint style="warning" %}
+Note that the following examples sets sensitive data as plain text. This is for education purpose only, it's highly recommended to use [secrets](../../building-applications/secrets.md).
```
Comment (eolivelli) [inline configuration-resources/large-language-models-llms/vertex-configuration.md:6]: do no write this, it is better to write good examples that people can copy/paste

File: get-started.md [hunk 4]
```diff
@@ -9,8 +9,7 @@ This guide uses Docker to run all the components of LangStream locally. If you w
 1. Install the LangStream CLI:
 
 ```
-brew tap LangStream/langstream
```
Comment (eolivelli) [inline get-started.md:12]: what's the difference ?

the new syntax handle upgrades ? 

currently I am using this flow:

```
brew update
brew upgrade langstream
```

File: installation/get-started-minikube.md [hunk 5]
```diff
@@ -1,71 +1,74 @@
...
-1. Port forward minio in a separate terminal:
+## Start the cluster
 
-```
-kubectl port-forward pod/minio 9000:9090 -n minio-dev  
+mini-langstream will do all the setup for you, in particular:
+- it starts `minikube` in a dedicated context
+- it deploys LangStream components using `helm`
+- it runs a stateful Kafka broker as docker container
+- it runs a stateful s3-compatible storage (MinIO) as docker container
```
Comment (eolivelli) [inline installation/get-started-minikube.md:40]: we should add the database here

File: pipeline-agents/ai-actions/ai-chat-completions.md [hunk 6]
```diff
@@ -14,7 +14,60 @@ layout:
...
+
+```yaml
+pipeline:
+  - name: "ai-chat-completions"
+    type: "ai-chat-completions"
+    configuration:
+      model: "gpt2"
+      messages:
+        - role: user
+          content: "You are a helpful assistant. Below you can find a question from the user. Please try to help them the best way you can.\n\n{{% value.question}}"
```
Comment (eolivelli) [inline pipeline-agents/ai-actions/ai-chat-completions.md:68]: I don't think that this works, IIRC you have to add a token with the MARK and HG simply gives you the word to replace the MARK

**Review by eolivelli — CHANGES_REQUESTED** [review]
Good work,
I have left some feedback PTAL

File: installation/get-started-minikube.md (hunk 5 above)
Comment (nicoloboschi) [reply installation/get-started-minikube.md:40]: it doesn't start herddb at the moment, we can add it

File: get-started.md (hunk 4 above)
Comment (nicoloboschi) [reply get-started.md:12]: this one is one-liner and it automatically handles updates.. 

I think it's compatible from the previous command

File: configuration-resources/large-language-models-llms/vertex-configuration.md (hunk 3 above)
Comment (nicoloboschi) [reply configuration-resources/large-language-models-llms/vertex-configuration.md:6]: the problem is that you have to fill the secrets and it's better to let the user know what you need to put in the secrets

I can update the examples to use the secrets and fill the secrets with the values

File: SUMMARY.md (hunk 2 above)
Comment (nicoloboschi) [reply SUMMARY.md:51]: right

**Review by eolivelli — APPROVED** [review]
LGTM

**Pull request merged**